File: jvm/cucumber-blueprint-jvm-example/src/test/java/cucumber/blueprint/jvm/example/integration/cucumber/scan/ExampleOsgiServiceITest.java
package cucumber.blueprint.jvm.example.integration.cucumber.scan;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(features = "classpath:features/exampleosgiservice.feature", glue = {
        "cucumber.blueprint.jvm.example.integration.cucumber.step.exampleosgiservice"}, strict = true)
public class ExampleOsgiServiceITest {
}

File: scala/cucumber-blueprint-scala-example/src/test/scala/cucumber/blueprint/jvm/example/integration/cucumber/scan/ExampleOsgiServiceITest.scala
package cucumber.blueprint.jvm.example.integration.cucumber.scan

import org.junit.runner.RunWith
import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber

@RunWith(classOf[Cucumber])
@CucumberOptions(features = Array("classpath:features/exampleosgiservice.feature"), glue = Array("cucumber.blueprint.jvm.example.integration.cucumber.scan.exampleosgiservice"), strict = true)
class ExampleOsgiServiceITest
